lib/Doctrine/Migrations/Tools/Console/Command/MigrateCommand.php

```diff
@@ -218,11 +218,16 @@ private function errorForAlias(string $versionAlias, bool $allowNoMigration) : i
         if (in_array($versionAlias, ['first', 'next', 'latest'], true) || strpos($versionAlias, 'current') === 0) {
             $version = $this->getDependencyFactory()->getVersionAliasResolver()->resolveVersionAlias('current');
 
-            $message = sprintf(
-                'The version "%s" couldn\'t be reached, you are at version "%s"',
-                $versionAlias,
-                (string) $version
-            );
+            // Allow meaningful message when latest version already reached.
+            if ($versionAlias === 'next' || $versionAlias === 'latest') {
+                $message = 'Already at latest version';
```

Comment:
What about adding info on the current version ?

Like `Already at latest version (%s)`